airflow/migrations/versions/0129_2_8_0_add_owner_display_name_to_audit_log_table.py

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Add owner_display_name to (Audit) Log table
+
+Revision ID: ede6abbcfe8e
+Revises: 405de8318b3a
+Create Date: 2023-09-01 15:10:07.942605
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "ede6abbcfe8e"
+down_revision = "405de8318b3a"
+branch_labels = None
+depends_on = None
+airflow_version = "2.8.0"
+
+TABLE_NAME = "log"
+
+
+def upgrade():
+    """Adds owner_display_name column to log"""
+    with op.batch_alter_table(TABLE_NAME) as batch_op:
+        batch_op.add_column(sa.Column("owner_display_name", sa.String(500)))
```

Comment:
Is this field ever NULL? Can we make the default an empty string instead?

Reply:
Agreed, set this to be serverside `""` and rebased.